src/boot/efi/stub.c

```diff
@@ -130,6 +130,53 @@ static void export_variables(EFI_LOADED_IMAGE_PROTOCOL *loaded_image) {
         (void) efivar_set_uint64_le(LOADER_GUID, L"StubFeatures", stub_features, 0);
 }
 
+static bool use_load_options(
+                EFI_HANDLE stub_image,
+                EFI_LOADED_IMAGE_PROTOCOL *loaded_image,
+                bool have_cmdline,
+                char16_t **ret) {
+
+        assert(stub_image);
+        assert(loaded_image);
+        assert(ret);
+
+        /* We only allow custom command lines if we aren't in secure boot or if no cmdline was backed into
```

Comment:
backed → baked ?

The comment says "or" but the check below says "and".

Reply:
You have your boolean logic confused: The comment states the intention correctly, but the if statement has to be inverted here to that effect. We want to reject the provided command line if sb is active *and* a command line was provided.